src/markdown-pages/docs/writing-own-tests/running-test-modes.md

```diff
@@ -0,0 +1,92 @@
+---
+path: "/docs/writing-own-tests/running-test-modes"
+title: "Running a Galasa test"
+---
+
+There are three modes in which you can run a Galasa test:<br><br>
+-  locally, with everything running on the local machine<br>
+-  locally but using a shared configuration that is hosted by the Galasa ecosystem<br>
+-  remotely, by submitting the test to run in the Galasa ecosystem<br> 
+
+
+The mode in which you choose to run a test depends on what you are trying to achieve. Use the following information to understand which mode is most appropriate for a given scenario. 
+
+
+## Running a test locally
+
+
+When you run a test locally, without using shared configuration, everything runs on your local machine. The Galasa bootstrap file is blank and makes no reference to an ecosystem. The Galasa framework is launched within the JVM on the local machine and the local file system holds all the configuration that is used by the test. The test runs in the local JVM and all test results and artifacts are stored on the local disk. 
+
+![running in local mode:](running-local.svg)
+
+You can run a test in this mode by using the following command:
+
+```
+galasactl runs submit --local … <more options are needed>
+```
```

Comment:
should we reference the syntax page for `runs submit local`

Also, it's not `--local` it's just `local`